cni/deployments/kubernetes/install/test/install_cni.go

```diff
@@ -224,9 +224,24 @@ func checkBinDir(t *testing.T, tempCNIBinDir, op string, files ...string) {
 	}
 }
 
+// checkTempFilesCleaned verifies that all temporary files have been cleaned up
+func checkTempFilesCleaned(tempCNIConfDir string, t *testing.T) {
+	files, err := ioutil.ReadDir(tempCNIConfDir)
+	if err != nil {
+		t.Fatalf("Failed to list files, err: %v", err)
+	}
+	for _, f := range files {
+		if strings.Contains(f.Name(), ".tmp") {
+			t.Fatalf("FAIL: Temporary file not cleaned in %v: %v", tempCNIConfDir, f.Name())
```

Comment:
```suggestion
			t.Fatalf("Temporary file not cleaned in %v: %v", tempCNIConfDir, f.Name())
```

Reply:
The other tests in this file have the FAIL prefix, so I was just following the convention. Should I remove it from all of the other tests in that file? It does help locate the exact error log when looking at the test logs